app/Services/User/LoginUserService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Services\User;
+
+use App\Http\Requests\Api\Auth\LoginRequest;
+use App\Http\Resources\ErrorResource;
+use App\Http\Resources\SuccessResource;
+use App\Repositories\UserRepository;
+use Exception;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Hash;
+
+class LoginUserService
+{
+    protected $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    public function execute(LoginRequest $request)
+    {
+        $user = $this->userRepository->findWhere(['email' => $request->input('email')])->first();
+
+        if (!$user || !$this->checkPassword($request->input('password'), $user->password)) {
+            return response([
+                'message' => ['These credentials do not match our records.']
+            ], 404);
+        }
+
+        try {
+            $token = $user->createToken('authToken')->plainTextToken;
+            return new SuccessResource(Response::HTTP_CREATED, "Logged In successfully.", [
+                'user' => $user,
```

Comment:
We don't have to send all the $user object. It contains many unnecessary data like created_at, updated_at, verified_at, ...etc.

Reply:
@aminetiyal I agree with @RanaEmad  you can create user resource for sending back the user data if you need.

Reply:
resolved [here](https://github.com/BuildForSDGCohort2/Team-248-Backend/pull/29/commits/af155184041cc39acd7da999f7ef5caaff672884)